docs/os/Manu.rst.include

```diff
@@ -0,0 +1,340 @@
+        :unix:
+
+        -   value: mkdir NewPythonFolder
+            type: input
+            prompt: 'username@linux~$'
+        -   value: ls
+            type: input
+            prompt: 'username@linux~$'     
+        -   'username@linux~$'
+
+
+
+After these two simple steps you should get a list as you did before when you learned how to use ``ls``. 
+In this list you should be able to see the 'NewPythonFolder' you have just created.
+
+6. How can I delete a folder or a file using a Terminal? (rm and rm -r)
+=============================
+To eliminate a folder you can use ``rm`` to eliminate a file or ``rm -r`` to eliminate a folder using the Terminal. 
+In this exercise you will eliminate the folder you created in the previous exercise. 
+
+1-Type and enter in your Terminal:
+   .. code-block:: 
+
+      rm -r NewPythonFolder
+
+2-Type and enter in your Terminal:
+   .. code-block:: 
+
+      ls
+
+This should look like this in your **Terminal**:
+
+
+.. tab::  Windows (Powershell) 
+
+   
+   .. termynal:: termynal:rmPS
+        :title: Powershell
+        :windows:
+
+        -   value: rm -r NewPythonFolder
+            type: input
+            prompt: 'PS C:\User\youruser>'
+        -   value: ls
+            type: input
+            prompt: 'PS C:\User\youruser>'
+        -   'PS C:\User\youruser>'  
+
+
+
+.. tab:: Windows (Command prompt)
+
+   
+   .. termynal:: termynal:rmCMD
+        :title: Command prompt
+        :windows:
+
+        -   value: rm -r NewPythonFolder
+            type: input
+            prompt: 'C:\User\youruser>'
+        -   value: ls
+            type: input
+            prompt: 'C:\User\youruser>'   
+        -   'C:\User\youruser>'
+
+
+
+.. tab:: Mac Terminal
+
+   
+   .. termynal:: termynal:rmMAC
+        :title: Command prompt
+        :unix:
+
+        -   value: rm -r NewPythonFolder 
+            type: input
+            prompt: 'username@mac ~ %'
+        -   value: ls
+            type: input
+            prompt: 'username@mac ~ %' 
+        -   'username@mac ~ %'
+
+
+   .. $ pwd  # On Unix-based systems (Linux or macOS)
+
+.. tab:: Linux Terminal
+
+   
+   .. termynal:: termynal:rmLINUX
+        :title: Command prompt
+        :unix:
+
+        -   value: rm -r NewPythonFolder
+            type: input
+            prompt: 'username@linux~$'
+        -   value: ls
+            type: input
+            prompt: 'username@linux~$'     
+        -   'username@linux~$'
+
+
+
+After these two simple steps you should get a list as you did before when you learned how to use ``ls``. 
+In this list you should not be able to see the 'NewPythonFolder' you have just deleted.
+
+7. Which Python version? (python --version or python3 --version)
+==================================
+
+Check which Python version you have.  **Note that this might change depending on what system you 
+have and wether you have more than one Python version installed**
+
+You will have to run either:
+   .. code-block:: 
```

Comment:
When you want more in one, I would do a comment:

```
.. code-block::
     python
     # or 
     python3
```
it is more compressed, and more readable.